easybuild/easyconfigs/l/libxc/libxc-3.0.0-intel-2017a.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'libxc'
+version = '3.0.0'
+
+homepage = 'http://www.tddft.org/programs/octopus/wiki/index.php/Libxc'
+description = """Libxc is a library of exchange-correlation functionals for density-functional theory.
+ The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+# Results for some functionals (e.g. mgga_c_tpss) deviate with too aggressive optimization settings.
+toolchainopts = {'lowopt': True}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.tddft.org/programs/octopus/down.php?file=libxc/']
+
+configopts = 'FC="$F77" FCFLAGS="$FFLAGS" --enable-shared --enable-fortran'
+
+runtest = 'check'
```

Comment:
@migueldiascosta Can you open a separate PR to enable the tests in the existing libxc 3.0.0 easyconfigs?